servicetest/filesystem/test_filesystem.py

```diff
@@ -0,0 +1,95 @@
+# Copyright (C) 2016 Pelagicore AB
+#
+# Permission to use, copy, modify, and/or distribute this software for
+# any purpose with or without fee is hereby granted, provided that the
+# above copyright notice and this permission notice appear in all copies.
+#
+# THE SOFTWARE IS PROVIDED "AS IS" AND THE AUTHOR DISCLAIMS ALL
+# WARRANTIES WITH REGARD TO THIS SOFTWARE INCLUDING ALL IMPLIED
+# WARRANTIES OF MERCHANTABILITY AND FITNESS. IN NO EVENT SHALL THE AUTHOR
+# BE LIABLE FOR ANY SPECIAL, DIRECT, INDIRECT, OR CONSEQUENTIAL DAMAGES
+# OR ANY DAMAGES WHATSOEVER RESULTING FROM LOSS OF USE, DATA OR PROFITS,
+# WHETHER IN AN ACTION OF CONTRACT, NEGLIGENCE OR OTHER TORTIOUS ACTION,
+# ARISING OUT OF OR IN CONNECTION WITH THE USE OR PERFORMANCE OF THIS
+# SOFTWARE.
+#
+# For further information see LICENSE
+
+import pytest
+
+import os
+import time
+import subprocess
+import fileapp
+import time
+
+from testframework import Container
+
+CURRENT_DIR = os.path.dirname(os.path.realpath(__file__))
+
+
+# This function is used by the 'agent' fixture to know where the log should
+# be stored
+def logfile_path():
+    return CURRENT_DIR + "/test.log"
+
+
+DATA = {
+    Container.PREFIX: "dbus-test-",
+    Container.CONFIG: '[{"enableWriteBuffer": false}]',
+    Container.BIND_MOUNT_DIR: "app",
+    Container.HOST_PATH: CURRENT_DIR
+}
+
+
+@pytest.mark.usefixtures("dbus_launch", "agent", "assert_no_proxy")
+class TestFileSystem(object):
+    """ This suite should do some basic testing of the Filesystem within the
+        containers and that the whole chain is working properly.
+    """
+
+    def test_enable_write_buffer_flag(self):
```

Comment:
Does this suite mean that we can remove the writeBuffer flag test from the dbus test suite?